source/postprocess/visualization/melt_fraction.cc

```diff
@@ -322,7 +322,10 @@ namespace aspect
                                                   "default of 15\\%. "
                                                   "The corresponding p-T-diagrams can be generated by running "
                                                   "the tests melt\\_postprocessor\\_peridotite and "
-                                                  "melt\\_postprocessor\\_pyroxenite.")
+                                                  "melt\\_postprocessor\\_pyroxenite."
```

Comment:
Please change this postprocessor to use the material model if possible. This prevents confusion about differently computed melt fractions.